robotoff/workers/tasks/import_image.py

```diff
@@ -47,37 +68,95 @@ def run_import_image_job(
         return
 
     with db:
-        with db.atomic():
-            save_image(barcode, source_image, product, server_domain)
-            import_insights_from_image(
-                barcode, image, source_image, ocr_url, server_domain
-            )
-        with db.atomic():
-            # Launch object detection in a new SQL transaction
-            run_object_detection(barcode, image, source_image, server_domain)
+        save_image(barcode, source_image, product, server_domain)
+
+    enqueue_job(
+        import_insights_from_image,
+        high_queue,
+        barcode=barcode,
+        image_url=image_url,
+        ocr_url=ocr_url,
+        server_domain=server_domain,
+    )
+    enqueue_job(
+        run_logo_object_detection,
+        high_queue,
+        barcode=barcode,
+        image_url=image_url,
+        server_domain=server_domain,
+    )
+    enqueue_job(
+        run_nutrition_table_object_detection,
+        high_queue,
+        barcode=barcode,
+        image_url=image_url,
+        server_domain=server_domain,
+    )
```

Comment:
(not sure) Don't we add more chance of having conflicts ? 
Shan't we instead have some dependency between them ?

Reply:
I added the following comment in the code:
```
    # The two following tasks take longer than the previous one, so it
    # shouldn't be an issue to launch tasks concurrently (and we still have
    # the insight import lock to avoid concurrent insight import in DB for the
    # same product)
```

Reply:
And it's interesting to launch them concurrently so that we get logo/nutriscore predictions faster.